Test/SqlDom/TSqlParserTest.cs

```diff
@@ -47,7 +47,7 @@ public void VerifySqlVersionToParserMapping()
                     Assert.Fail("New SqlVersion {0} needs to be added to knownVersionMapping.", version.ToString());
                 }
 
-                TSqlParser versionedParser = parser.Create(version, false);
+                TSqlParser versionedParser = TSqlParser.CreateParser(version, false);
```

Comment:
Can we keep the old code and add a new line for the static version of the create?